<ui version="4.0" >
 <class>EditWidgetClassificationObject</class>
 <widget class="QWidget" name="EditWidgetClassificationObject" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>EditWidgetClassificationObject</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBoxPreview" >
     <property name="title" >
      <string>Preview</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="blissart::SamplesPreviewWidget" native="1" name="svWidget" >
        <property name="minimumSize" >
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Select labels:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy" >
      <cstring>lwLabels</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QListWidget" name="lwLabels" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>blissart::SamplesPreviewWidget</class>
   <extends>QWidget</extends>
   <header>SamplesPreviewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lwLabels</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
